test/unit/helpers/cache.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+var chai = require('chai');
+var expect = require('chai').expect;
+var sinon = require('sinon');
+var redis = require('redis');
+
+var cache = require('../../../helpers/cache');
+
+describe('cache', function () {
+
+	it('is not enabled', function () {
+		cache.connect(false, {}, {}, function (err, result) {
+			expect(result.cacheEnabled).to.eq( false );
+			expect(result.client).to.be.null;
+		});
+	});
+
+	it('is enabled redis client callback ready', function () {
+
+		var logger = {
```

Comment:
logger and mockredisclient are duplicated in each test, it'd be nicer if you declare them in the describe scope and use the same object in each test.
Also, update `mockredisclient` variable name to camel case.

Reply:
No problem. Moved the logger up to the describe scope.